### PR TITLE
Straight forward "be excellent" script

### PR DESCRIPTION
**Pull request by bsutt123**
Really just a simple script where if it hears "be excellent" (with word boundaries at both ends and any one white space not new line in the middle) it will respond with the "be excellent to each other" gif from bill and teds excellent adventure.

### CODE REVIEW

File: scripts/be-excellent.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = (robot) ->
```
Comment (sukima) [inline scripts/be-excellent.coffee:16]: Missing [documentation](https://hubot.github.com/docs/scripting/#documenting-scripts):

```js
// Description:
//   Bill and Ted's proclamation.
//
// Dependencies:
//   NONE
//
// Configuration:
//   NONE
//
// Commands:
//   be excellent - Shows an image of Bill and Ted
//
// Notes:
//   NONE
//
// Author:
//   @bsutt123
```

File: scripts/be-excellent.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+  robot.hear /\bbe\sexcellent\b/, (msg) ->
```
Comment (sukima) [inline scripts/be-excellent.coffee]: Do we want `/.../i` to ignore case?

**Review by sukima — APPROVED** [review]

File: scripts/be-excellent.coffee (hunk 2 above)
Comment (jnimety) [reply scripts/be-excellent.coffee]: and maybe `\s+` to allow for more whitespace

Comment (bsutt123) [reply scripts/be-excellent.coffee]: I'll throw that and documentation in real quick and push it up

Comment (bsutt123) [reply scripts/be-excellent.coffee]: Done and Done on the white space. I took a look at other scripts for whitespace and it seemed like static was what people went with, but I could go either way on it.

**Review by sukima — APPROVED** [review]

**Pull request merged**